src/Utilities/Workloads.js

```diff
@@ -144,3 +145,48 @@ export const pruneWorkloadsRulesFilters = (localFilters, filterCategories) => {
     return arr;
   }, []);
 };
+
+export const switchSort = (sortIndex, item) => {
+  const rule = item[0]?.rule; // Using optional chaining to check if 'rule' is defined
```

Comment:
nit-pick: what are the circumstances when the rule can be not defined? are they expected/anticipated? if not, then there is no reason to introduce optional chaining.